docs/sources/json.md

```diff
@@ -44,6 +44,31 @@ Note that:
 * For list we define a key with its index and they can nest other values.
 * All `Key`s are lowercase so this source ignores uppercase properties.
 
+### Magic `_self` key
```

Comment:
hahaha even tho I like the "Magic" I think it might be an offputing name for someone using the library 😅 . What about **Special** _self key?